ConnectorArchicad/ConnectorArchicad/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "ConnectorArchicad": {
+      "commandName": "Project",
+      "commandLineArgs": "19723 888"
```

Comment:
What are these obscure arguments 😅 Could we get some comments to detail what they mean?

Reply:
The first argument is the HTTP port of the Archicad JSON API, and the second is the Archicad version number (for testing purposes I use very distinct numbers). I'll remove this file from the PR.

Reply:
Beautiful, i'll wait for this removal and aprove+merge